src/assets/locales/en.json

```diff
@@ -2671,14 +2671,17 @@
         }
       },
       "daoSettingsInfo": {
+        "address": "Address",
+        "blockchain": "Blockchain",
         "chain": "Chain",
-        "ens": "ENS name",
-        "links": "Resources",
+        "ens": "ENS Name",
```

Comment:
I don't agree but we actually have a pattern of not capitalizing all words in a title/label/button etc
```suggestion
        "ens": "ENS name",
```